data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/ResponseCardinality.java

```diff
@@ -0,0 +1,36 @@
+package org.opensearch.dataprepper.plugins.lambda.processor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ResponseCardinality {
+    STRICT("strict"),
+    AGGREGATE("aggregate");
+
+    private final String value;
+
+    ResponseCardinality(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    private static final Map<String, ResponseCardinality> RESPONSE_CARDINALITY_MAP = new HashMap<>();
+
+    static {
+        for (ResponseCardinality type : ResponseCardinality.values()) {
+            RESPONSE_CARDINALITY_MAP.put(type.getValue().toLowerCase(), type);
+        }
+    }
+
+    // Default value is STRICT
```

Comment:
```suggestion
    @JsonCreator
```

This will work better with schema validation.